index.js

```diff
@@ -66,6 +66,10 @@ module.exports = (dest, opts) => {
 		throw new gutil.PluginError('gulp-changed', '`dest` required');
 	}
 
+	if (opts.transformPath !== undefined && typeof opts.transformPath !== 'function') {
```

Comment:
As commented in https://github.com/sindresorhus/gulp-changed/pull/60#discussion_r101000468, the first check is unnessecary.